client/components/Overview/components/Sizes.jsx

```diff
@@ -4,19 +4,31 @@ import React from 'react';
 import SizesRender from './SizesRender';
 
 //
-const Sizes = ({ currentStyle, setCurrentSize, setCurrentQuantity }) => {
+const Sizes = ({
+  currentStyle, setCurrentSize, setCurrentQuantity, currentSize,
+  addToCartNoSize, setAddToCartNoSize, value, setValue
+}) => {
   //  for loop to make the sku property an array rather than multiple objects
   //  this makes it able to be mapped in SizesRender function
   const styleArray = [];
   for (const key in currentStyle.skus) {
-    styleArray.push(currentStyle.skus[key]);
+    if (currentStyle.skus[key].quantity === 0) {
+      styleArray.push({ size: 'OUT OF STOCK' });
+    } else {
+      styleArray.push(currentStyle.skus[key]);
+    }
   }
 
   return (
     <SizesRender
       styleArray={styleArray}
```

Comment:
Opinion: would be easier to read if your props on lines 27-31 are ordered in the same way as lines 7-9. Doesn't affect anything, but it makes it easier to review

Reply:
I will make a ticket to fix that.